components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
The issue with this code is definitely maintainability. If you ask me, all regex is evil, unmaintainable (even by the person who wrote it), and spooky. It looks like what you're doing is modifying the DOM structure, by first doing a `el.innerHTML()`, and then nuking parts of the string with regex. Why not just pick the elements you want to remove with selectors?